dhis-2/dhis-support/dhis-support-db-migration/src/main/java/org/hisp/dhis/db/migration/v33/V2_33_5__Update_job_parameters_with_system_setting_values.java

```diff
@@ -126,11 +126,15 @@ public void migrate( final Context context ) throws Exception
                 {
                     while ( rs.next())
                     {
-                        TrackerProgramsDataSynchronizationJobParameters jobparams = reader
-                            .readValue( rs.getString( "jsonbjobparameters" ) );
-                        jobparams.setPageSize( trackerPageSize );
-
-                        updatedJobParameters.put( rs.getInt( "jobconfigurationid" ), jobparams );
```

Comment:
Unfortunately we are not allowed to change released upgrade routines. Flyway stores a checksum of the migration and will complain if it is changed after applied to a database.